### PR TITLE
fix issue #648 , service-binding-operator panics if there is a configMapRef in envFrom

### PR DESCRIPTION
**Pull request by qibobo**
To fix issue #648

### CODE REVIEW

File: pkg/controller/servicebindingrequest/binder_test.go [hunk 1]
```diff
@@ -171,6 +171,35 @@ func TestBinderNew(t *testing.T) {
...
+			ConfigMapRef: &corev1.ConfigMapEnvSource{
+				LocalObjectReference: corev1.LocalObjectReference{
+					Name: "test-configmap",
+				},
+			},
+		})
+
+		list := binder.appendEnvFrom(envFrom, secretName)
+		require.Equal(t, 2, len(list))
+		require.Equal(t, secretName, list[1].SecretRef.Name)
```
Comment (DhritiShikhar) [inline pkg/controller/servicebindingrequest/binder_test.go:197]: There are two items in `list` -> `require.Equal(t, 2, len(list))`

We should also have a `require.Equal` check for for `list[0]`

**Comment by qibobo** [comment]
@DhritiShikhar fixed.

**Pull request merged**